contrail-introspect-cli/main.go

```diff
@@ -111,6 +111,22 @@ func DescRiSummary() DescCollection {
 	}
 }
 
+func DescCtrlRouteSummary() DescCollection {
+    return DescCollection{
+        PageArgs: []string{"controller-fqdn", "search"},
+		PageBuilder: func(args []string) Sourcer {
+			path := fmt.Sprintf("Snh_ShowRouteSummaryReq?search_string=%s", args[1])
+			return Webui{Path: path, VrouterUrl: args[0], Port: 8083}
+		},
+		BaseXpath: "ShowRouteSummaryResp/tables",
```

Comment:
The baseXpath is the path to reach the collection elements:
ShowRouteSummaryResp/tables/list